hazelcast/src/main/java/com/hazelcast/map/merge/LatestUpdateMapMergePolicy.java

```diff
@@ -25,10 +25,10 @@
 import java.io.IOException;
 
 /**
- * LatestUpdateMapMergePolicy causes the merging entry to be merged from source to destination map
- * if source entry has updated more recently than the destination entry.
- *
- * This policy can only be used of the clocks of the machines are in sync.
+ * Merges map entries from source to destination map if the source entry
+ * was updated more frequently than the destination entry.
```

Comment:
`updated more frequently`? I would then expect this policy to use the entry which was updated more times, even though the last update was perhaps less recent than the other entry.

Reply:
Changed this to `updated more recently`, good catch!